canvas_modules/common-canvas/src/tooltip/tooltip.scss

```diff
@@ -74,8 +74,8 @@
 
 .common-canvas-tooltip .tipArrow {
 	position: absolute;
-	width: 10px;
-	height: 14px;
+	width: $spacing-04;
+	height: $spacing-05;
```

Comment:
This needs to be reverted back since it requires a specific width and height
<img width="354" alt="image" src="https://github.com/user-attachments/assets/ca655c12-e1a4-428f-b8db-f8a70d3de308">
vs master:
<img width="348" alt="image" src="https://github.com/user-attachments/assets/897b308a-3fac-4e26-849b-ab00501274ea">